mthamayor/auto-mart
# Created user signup API endpoint [Finishes #166107231]

**What this PR does**
Creation of API which enables potential users to signup

**Description**
Enables potential users to create an account by inserting the following request body in the provided form: first name, last name, address, password, and email

**Testing endpoint procedures**
1. Install dependencies with
`npm install`
2. Start up the local development server with
`npm run dev`
 3. Access the endpoint API at URL:  **http://localhost:3001/api/v1/auth/signup**  on Postman POST request.
Fill up the form body to contain the following
![Screenshot (110)](https://user-images.githubusercontent.com/16121915/58107701-4ad7e900-7be2-11e9-8364-1abf721605ad.png)
it should return a success response
![Screenshot (111)](https://user-images.githubusercontent.com/16121915/58107721-56c3ab00-7be2-11e9-9fd4-d8df95f4a681.png)

Pivotal tracker story
https://www.pivotaltracker.com/story/show/166107231
